src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/TypedArray.cs

```diff
@@ -208,7 +208,7 @@ public unsafe int CopyFrom(ReadOnlySpan<U> span)
             // source has to be instantiated.
             if (span == null || span.Length == 0)
             {
-                throw new System.ArgumentException($"Invalid argument: {nameof(span)} can not be null and must have a length");
+                throw new System.ArgumentException(SR.Format(SR.ArgumentCannotBeNullWithLength, nameof(span)));
```

Comment:
No need to fix in this PR, but perhaps we should consider an analyzer that spots when we're not passing a parameter name to an ArgumentException constructor. ie
```c#
throw new System.ArgumentException(SR.Format(SR.ArgumentCannotBeNullWithLength, nameof(span)), nameof(span));
```
although since the parameter is appended to the message anyway, this is better 
```c#
throw new System.ArgumentException(SR.ArgumentCannotBeNullWithLengthNoParameterInThisMessage, nameof(span));
```
@buyaa-n just curious, was this considered in designing your analyzer? https://github.com/dotnet/roslyn-analyzers/pull/3500 ..maybe too noisy?

Reply:
That analyzer only analyze if `string paramName` parameter is passed correctly, does not account `string message` or any other parameter of   `ArgumentException (string message, string paramName);` constructor overloads. I think checking if message includes parameter name will be quite noisy

Reply:
@buyaa-n I was thinking of a rule something like "if message is passed an expression containing nameof an argument, but no argument is passed as a parameter.." but I guess that is relatively unlikely  so never mind.